package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.26.7",
+  "version": "0.26.11",
```

Comment:
No need to bump this, but we should also see if we can remove this package.json file and anything bad happens (not on what gets pushed to main!).